projects/llm-semconv.md

```diff
@@ -0,0 +1,72 @@
+# LLM Observability Semantic Conventions
+
+## Description
+
+This project aims to establish observability standards for Large Language Model (LLM) applications, focusing on API interactions and multi-step LLM frameworks. Key areas include:
+
+- Developing semantic conventions for standard and complex LLM interactions.
+- Initiating vendor-specific conventions as a foundation for broader contributions.
+- Concentrating on attributes relevant to Metrics and Spans.
+
+For Spans: Instrumentation and debugging guidance for multi-step LLM applications, with recommendations on user privacy and intellectual property security, marking sensitive attributes and events as opt-in.
+
+For Metrics: Tracking model performance by collecting and correlating data like model names, versions, parameters, latency, and evaluation scores.
+
+## Deliverables
+
+- Experimental semantic conventions for LLMs, allowing flexibility for evolution in this fast-paced field.
+- Reference prototypes demonstrating the application of these conventions in data collection, processing, and analysis.
+- Guidelines for monitoring LLM applications, including implementation strategies and privacy/security considerations.
+
+## Staffing / Help Wanted
+
+We will gather further staff as the project is established. We need to 
+make sure there is enough representation across both vendors and customers
+who will benefit from these semantic conventions.
+
+**Project Leads:**
+
+- @drewby
+- @nirga
+
+**Sponsoring Members:**
+
+- @lmolkova
+- *Seeking additional sponsors*
+
+**Engineers:**
+
+- @nirga
+- @sudivate
+- *Positions open for more engineers*
+
+**Maintainers:**
+
+- To be determined
+
+## Meeting Times
+
+Alternating weekly meetings to accommodate different time zones:
+
+- Week 1: Mondays at 8 AM PST
+- Week 2: Mondays at 4 PM PST
+
+## Timeline
+
+1. Initial Draft: February 2024
+2. Review and Refinement: April 2024
```

Comment:
Can  we add prototypes somewhere between April/June?

I'm volunteering to help prototype Azure OpenAI SDKs in .NET and/or Java, it'd be great to have prototype in Python

Reply:
For Python we could perhaps lean on OpenLLMetry? https://github.com/traceloop/openllmetry

I think it'll also be important to prototype something that uses OpenAI's Node SDK and/or event have a runtime-agnostic instrumentation package. JS is a quiet-but-big part of OpenAI usage from my standpoint.

Reply:
Added this to the timeline